### ISSUE TITLE
Loss rate is not decreasing

### ISSUE THREAD

**Issue by ayush-angelium** [Heonozis/FaceShifter-pytorch#11]
Hello @by321 @Heonozis ,
when we train this model on the given dataset then the loss-rate is not decreasing from 5.
So please suggest me how can we train the model for better results.

Thanks
